### ISSUE TITLE
The experimental environment

### ISSUE THREAD

**Issue by Caowintay** [kmsiapps/e2e-dnn-comm-for-image#4]
Dear author,
I recently set up a lab environment and had some problems with the version, can you provide a specific version of the experiment? Such as the version of CUDNN, CUDA and Tensorflow. Thanks a lot.

**Comment by kmsiapps** [comment]
Can you explain more about the issue? I used the official tensorflow docker image (https://www.tensorflow.org/install/docker) in the Ubuntu 20.04 environment.

**Comment by Caowintay** [comment]
> Can you explain more about the issue? I used the official tensorflow docker image (https://www.tensorflow.org/install/docker) in the Ubuntu 20.04 environment.

Thanks a lot for your replying. I have run the code. But now I have a new problem. Could you give me some help?
The problem is:
I ran the code with Dataset CIFAR10, after training, I ran evaluate.py, but the results are not ideal. Such as followings.
The reconstructed image is black. The train and test loss is about 0.3.
![image](https://user-images.githubusercontent.com/46077683/203497203-f2040f72-4300-49eb-a720-2fdabd136397.png)

**Comment by kmsiapps** [comment]
That's weird. Are the image pixel values between 0 and 1? Inconsistent image pixel values may cause those problems.
Also, the loss value of 0.3 is too high. You may check the training procedure once again.

**Comment by Caowintay** [comment]
> That's weird. Are the image pixel values between 0 and 1? Inconsistent image pixel values may cause those problems. Also, the loss value of 0.3 is too high. You may check the training procedure once again.

Ok,Thanks a lot. I will try again.
